napari/layers/labels/labels.py

```diff
@@ -664,7 +664,7 @@ def _ensure_int_labels(self, data):
                     )
                 )
             if data_level.dtype == bool:
-                int_data.append(data_level.astype(np.int8))
+                int_data.append(data_level.view(np.uint8))
```

Comment:
```suggestion
                int_data.append(data_level.astype(np.uint8))
```

View is view on original data so if bool will be stored in optimized way then we lost information

Reply:
@Czaki have you read the description of this PR? Or the test I added? The change to `view` is an intentional bug fix. I have never encountered an array library in Python that uses bit-packing for bool, have you?

Reply:
OK. I have even checked the documentation now and `Boolean (True or False) stored as a byte`. so it is clear states that it should be safe.